src/main/content/_assets/css/start.scss

```diff
@@ -677,6 +677,55 @@ html, body {
     margin-left: 24px;
 }
 
+@media (min-width: 576px) {
+    .modal-dialog {
+        max-width: 575px;
+    }  
+}
```

Comment:
A style suggestion is to move this `@media` into the `.modal-content` on line 686 since the `@media` only applies to the modal.  I don't have a strong preference and just wanted to mention the option since we are using Sass rather than vanilla CSS.

Reply:
done